chain/chunks/src/lib.rs

```diff
@@ -132,43 +137,64 @@ impl RequestPool {
     }
 }
 
+#[derive(Debug, Eq, PartialEq)]
+enum Seal<'a> {
+    Previous,
+    Active(&'a mut ActiveSealChallenge),
+}
+
 #[derive(Clone, Debug, Eq, PartialEq)]
-pub struct Seal {
+struct ActiveSealChallenge {
```

Comment:
**Challenge**: please try to avoid using a noun "Challenge" because this concept is already taken for another purpose.

Reply:
Renamed to demur (meaning "the action or process of objecting to or hesitating over something", which seemed appropriate because we are _hesitating_ to believe the chunk really exists unless we get a few random parts) https://github.com/nearprotocol/nearcore/pull/2702/commits/3e15b21347236f5d006d7574de928003f42b8ed8